content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,255 @@ weight = 20
 
 +++
 
+## What is Kubeflow?
```

Comment:
I think this looks better as H3, like the `How to install Kubeflow?` below it:

```suggestion
### What is Kubeflow?
```

Reply:
We might want to keep titles with the same font if it is not sub-title. Like in Architecture page: https://deploy-preview-3693--competent-brattain-de2d6d.netlify.app/docs/started/architecture/#conceptual-overview.
So all title fonts on the same level will be consistent.

So if you don't want to make `How to install Kubeflow? ` subtitle, I can change it as follows:
```
## How to install Kubeflow? 
```

Reply:
I will make them both H2. If we make the first title H3, website will show outline not accurate 
<img width="293" alt="Screenshot 2024-03-08 at 13 07 00" src="https://github.com/kubeflow/website/assets/31112157/9a4181c1-7b3c-4742-ae77-a26bd45b6252">